libc/test/UnitTest/HermeticTestUtils.cpp

```diff
@@ -136,7 +136,7 @@ void operator delete(void *) {
   __builtin_trap();
 }
 
-void operator delete(void *ptr, size_t size) { __builtin_trap(); }
+void operator delete(void */*ptr*/, size_t /*size*/) { __builtin_trap(); }
```

Comment:
```suggestion
void operator delete(void * /*ptr*/, size_t /*size*/) { __builtin_trap(); }
```
or did `clang-format` choose this?

Reply:
Might have been me, I'll try putting the space see if it complains

Reply:
Will be handle in the next PR that deals with unused parameters